docs/src/routes/docs/installation.svx

```diff
@@ -3,110 +3,380 @@ name: Installation
-    <CopyableCode>
+    <CopyableCode filename="webpack.config.js">
 
-```sh
-npm install --save-dev svelte-preprocess sass postcss
+```js
+module.exports = {
+  module: {
+    rules: [
+      // ...,
+      {
+        test: /attractions\\/index\.js$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env']
+          },
+        },
+      },
+    ],
+  },
+};
 ```
 
     </CopyableCode>
-  {:else if packageManager === packageManagers[1]}
-    <CopyableCode>
+  </div>
+{/if}
 
-```sh
-yarn add --dev svelte-preprocess sass postcss
+**Step 1.** Add Attractions {#if yourFramework !== Framework.other} and its peer dependencies {/if}
+to your {@html yourFramework === Framework.other ? '`dependencies`' : '`devDependencies`'}:
+
+{#if yourPackageManager === PackageManager.npm}
+  <CopyableCode terminal>
+    {#if yourFramework === Framework.other}
+
+```shell
+npm install --save attractions
 ```
 
-    </CopyableCode>
-  {:else}
-    <CopyableCode>
+    {:else}
 
-```sh
-pnpm add --save-dev svelte-preprocess sass postcss
+```shell
+npm install --save-dev attractions svelte-preprocess sass postcss
 ```
 
-    </CopyableCode>
+    {/if}
+  </CopyableCode>
+{:else if yourPackageManager === PackageManager.yarn}
+  <CopyableCode terminal>
+    {#if yourFramework === Framework.other}
+
+```shell
+yarn add attractions
+```
+
+    {:else}
+
+```shell
+yarn add --dev attractions svelte-preprocess sass postcss
+```
+
+    {/if}
+  </CopyableCode>
+{:else}
+  <CopyableCode terminal>
+    {#if yourFramework === Framework.other}
+
+```shell
+pnpm add attractions
+```
+
+    {:else}
+
+```shell
+pnpm add --save-dev attractions svelte-preprocess sass postcss
+```
+
+    {/if}
+  </CopyableCode>
+{/if}
+
+{#if yourFramework !== Framework.other}
+
+  **Step 2.** Add `svelte-preprocess` to your preprocessor chain
+  {#if yourBundler === Bundler.rollup}
+    {#if yourFramework === Framework.svelte}
+      ([as shown here](https://github.com/sveltejs/svelte-preprocess/blob/main/docs/usage.md#with-rollup-plugin-svelte)):
+    {:else}
+      ([as shown here](https://github.com/sveltejs/svelte-preprocess/blob/main/docs/usage.md#with-sapper)):
+    {/if}
+  {:else}
+    ([as shown here](https://github.com/sveltejs/svelte-preprocess/blob/main/docs/usage.md#with-svelte-loader)):
   {/if}
-</div>
 
-**Step 3.** Add `svelte-preprocess` to your preprocessor chain ([as shown here](https://github.com/sveltejs/svelte-preprocess/blob/main/docs/usage.md)):
+  <CopyableCode filename="svelte.config.js">
+
+```js
+const sveltePreprocess = require('svelte-preprocess');
+
+module.exports = {
+  preprocess: [sveltePreprocess()],
+};
+```
+
+  </CopyableCode>
+
+  {#if yourBundler === Bundler.rollup}
+    {#if yourFramework === Framework.svelte}
+      <CopyableCode filename="rollup.config.js">
+
+```js
+import svelte from 'rollup-plugin-svelte';
+import svelteConfig from './svelte.config.js';
```

Comment:
I don't think this works. You cannot mix ES6 modules with CommonJS modules. I know it is suggested in the docs of `svelte-preprocess`, but I never succeeded in actually importing the config.
I usually just leave the `svelte.config.js` file there for the Svelte VS Code extension to pick up on it.

Perhaps it will work with Webpack (as it uses CommonJS all the way), but not with Rollup (as `svelte.config.js` is required by the Svelte Language Server to be in CommonJS).

Reply:
Actually, I just tried it again now, and it magically worked. I thought we cannot mix ES6 and CommonJS imports, but it seems like Rollup has been updated or something and it is no longer complaining

Reply:
No, actually, it wasn't working, for various reasons, but now it's all good.